Delete Cartfile.resolved

johndpope opened the pull request:
this is causing resolution to older version / not latest. 

➜  frontpage-ios-app git:(master) ✗ carthage bootstrap --platform iOS
*** No Cartfile.resolved found, updating dependencies
*** Fetching apollo-ios
*** Fetching SQLite.swift
*** Checking out apollo-ios at "0.6.5" <- picking up correct version after Cartfile.resolved is deleted
*** Checking out SQLite.swift at "0.11.3"
*** xcodebuild output can be found in /var/folders/jz/lf_rj_ln1712tyf0pd9jq8f00000gn/T/carthage-xcodebuild.F6oyXh.log
*** Building scheme "SQLite iOS" in SQLite.xcodeproj
*** Building scheme "Apollo" in Apollo.xcworkspace

johndpope committed:
Delete Cartfile.resolved

apollo-cla commented:
@johndpope: Thank you for submitting a pull request! Before we can merge it, you'll need to sign the Meteor Contributor Agreement here: https://contribute.meteor.com/

johndpope commented:
I presumed getting latest version would solve problems. 
but the latest is not the alpha branches. 
so removing the resolved file does run into other problems - 
specifically - the latest (I realize there are versions 17) but carthage is finding 0.16
the v 0.16.0 apollo-codegen is missing some dependencies as below. 

git checkout v0.16.0


need a more solid way to tie  code-gen version with the apollo framework build script

The version of Apollo.framework in your project requires apollo-codegen 0.16, but 0.10.0 seems to be installed. Installing...


 npm install -g wrap-ansi
+ wrap-ansi@3.0.1
added 5 packages in 1.147s
➜  apollo-codegen git:(4f19e67) ✗ sudo npm install . -g
npm WARN checkPermissions Missing write access to /Users/johndpope/Documents/gitWorkspace/apollo-codegen/node_modules/wrap-ansi
npm WARN checkPermissions Missing write access to /Users/johndpope/Documents/gitWorkspace/apollo-codegen/node_modules/cliui
npm WARN checkPermissions Missing write access to /Users/johndpope/Documents/gitWorkspace/apollo-codegen/node_modules
npm ERR! path /Users/johndpope/Documents/gitWorkspace/apollo-codegen/node_modules/wrap-ansi
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall access
npm ERR! enoent ENOENT: no such file or directory, access '/Users/johndpope/Documents/gitWorkspace/apollo-codegen/node_modules/wrap-ansi'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/johndpope/.npm/_logs/2017-09-28T00_36_58_843Z-debug.log
➜  apollo-codegen git:(4f19e67) ✗ npm install wrap-ansi
+ wrap-ansi@3.0.1
added 48 packages in 2.858s
➜  apollo-codegen git:(4f19e67) ✗ sudo npm install . -g
npm WARN checkPermissions Missing write access to /Users/johndpope/Documents/gitWorkspace/apollo-codegen/node_modules/cliui
npm ERR! path /Users/johndpope/Documents/gitWorkspace/apollo-codegen/node_modules/cliui
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall access
npm ERR! enoent ENOENT: no such file or directory, access '/Users/johndpope/Documents/gitWorkspace/apollo-codegen/node_modules/cliui'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/johndpope/.npm/_logs/2017-09-28T00_37_15_591Z-debug.log
➜  apollo-codegen git:(4f19e67) ✗ npm install cliui
+ cliui@3.2.0
added 8 packages in 1.529s
➜  apollo-codegen git:(4f19e67) ✗ sudo npm install . -g
/usr/local/bin/apollo-codegen -> /usr/local/lib/node_modules/apollo-codegen/lib/cli.js
+ apollo-codegen@0.16.0
added 24 packages, removed 2 packages, updated 6 packages and moved 2 packages in 3.015s

johndpope commented:
thinking all the trouble is due to incorrect upstream repo 
should be 

github "apollographql/apollo-ios" not

github "apollostack/apollo-ios"

paulmelnikow commented:
Should the repo path be fixed instead of the file removed?

designatednerd commented:
I think when there's a `Cartfile.resolved`, Carthage. doesn't properly resolve the change from `apollostack` to `apollographql` at runtime. Merging this.
